CHANGELOG.rst

```diff
@@ -1,5 +1,61 @@
 Changelog
 =========
+1.13 (2020-12-22)
+-----------------
+MLflow 1.13 includes several major features and improvements:
+
+Features:
+
+New fluent APIs for logging in-memory objects as artifacts:
+
+- Add ``mlflow.log_text`` which logs text as an artifact (#3678, @harupy)
+- Add ``mlflow.log_dict`` which logs a dictionary as an artifact (#3685, @harupy)
+- Add ``mlflow.log_figure`` which logs a figure object as an artifact (#3707, @harupy)
+- Add ``mlflow.log_image`` which logs an image object as an artifact (#3728, @harupy)
+
+UI update / fixes (#3867, @smurching):
+
+- Add "Empty state" overlay for experiments
+- Add ``SectionErrorBoundary`` for better error handling
+- Use local storage to persist model registry UI settings
+- Add ``data-test-id`` / DOM classname to facilitate e2e UI testing
+- Add description to model version table
+- Add model version link in compact experiment table view
+- Add logged/registered model links in experiment runs page view
+- Enhance artifact viewer for MLflow models
+
+Autologging enhancements:
+
+- Add ``exclusive`` flag to remaining relevant autologging integrations (#3869, @dbczumar)
```

Comment:
```suggestion
- Add ``exclusive`` configuration flag to autologging integrations (#3869, @dbczumar)
```

Reply:
@harupy We're also introducing a `disabled` flag as part of the release, but I don't see that called out in the changelog. Did we miss the PR for it?

Reply:
@dbczumar I think it's because #3815 is marked as `rn/none`. I'll manually move it here.